tools/grafana/alerting/alerts.yml

```diff
@@ -6,6 +6,163 @@ groups:
     name: awx_rules
     orgId: 1
     rules:
+      - condition: B
+        dashboardUid: awx
+        data:
+          - refId: A
+            queryType: ''
+            relativeTimeRange:
+              from: 900
+              to: 0
+            datasourceUid: PBFA97CFB590B2093
```

Comment:
Where does this value come from? Will this change between different instances of Grafana?

Reply:
These Uid's are a bit mysterious. This feature of the "provisioning files" for alerting is relatively new so the docs are not great.

That said, we tested deleting all grafana containers and volumes and starting from scratch. The alerts come up and work, and appear to point to the correct data sources. I think we can leave this as it is and perhaps if we DO want to have control of what it is we need to set the Uid in tools/grafana/datasources/prometheus_source.yml